assembl/tests/views_tests/test_graphql.py

```diff
@@ -1938,3 +1940,153 @@ def __init__(self, filename, type):
     assert resources_center['headerImage'] is not None
     assert '/documents/' in resources_center['headerImage']['externalUrl']
     assert resources_center['headerImage']['title'] == 'new-img.png'
+
+
+def test_query_sections(discussion, graphql_request, sections):
+    from assembl.models.section import SectionTypesEnum
+    query = u"""
+query { sections {
+    id
+    title(lang:"en")
+    titleEntries {
+        localeCode
+        value
+    }
+    url
+    sectionType
+    order
+} }"""
+    res = schema.execute(query, context_value=graphql_request)
+    assert len(res.data['sections']) == 6
+    assert res.data['sections'][0]['title'] == u'Home'
+    assert res.data['sections'][0]['titleEntries'][0]['localeCode'] == u'en'
+    assert res.data['sections'][0]['titleEntries'][0]['value'] == u'Home'
+    assert res.data['sections'][0]['url'] is None
+    assert res.data['sections'][0]['sectionType'] == SectionTypesEnum.HOMEPAGE.value
+    assert res.data['sections'][0]['order'] == 0.0
+
+    assert res.data['sections'][-1]['url'] == u'http://www.gnu.org'
+    assert res.data['sections'][-1]['sectionType'] == SectionTypesEnum.CUSTOM.value
+    assert res.data['sections'][-1]['order'] == 4.0
+    assert res.data['sections'][-1]['title'] == u'GNU is not Unix'
+
+def test_mutation_create_section(sections, graphql_request):
+    from assembl.models.section import SectionTypesEnum
+    title_entries = [
+        {u"value": u"Section personnalisée", u"localeCode": u"fr"},
+        {u"value": u"Custom section", u"localeCode": u"en"}
+    ]
+    variables = {
+        "titleEntries": title_entries,
+        "url": u"http://www.example.com",
+        "order": 5.0
+    }
+    res = schema.execute(u"""
+mutation createSection($titleEntries:[LangStringEntryInput!]!,$url:String,$order:Float) {
```

Comment:
Can you please use the same styleguide as prettier to format query in Python tests? space after ',' and ':'. Thanks.

Reply:
done